src/routes/docs/advanced/security/mfa/+page.markdoc

```diff
@@ -0,0 +1,36 @@
+---
+layout: article
+title: Multi-factor Authentication
+description: Appwrite helps you secure your developer accounts with MFA (multi-factor authentication).
+---
+Multi-factor authentication (MFA) adds multiple layers of authentication to your Appwrite account,
+which means a malicious actor cannot gain access to your accounts, even if one of the factors is compromised.
```

Comment:
I think this wording from other file is better

```
When MFA is enabled, a malicious actor needs to compromise multiple authentication factors to gain unauthorized access.
```

I like `needs to compromise multiple authentication factors` over `even if one of the factors is compromised`

Reply:
Done